massa-graph/src/export_active_block.rs

```diff
@@ -58,7 +58,7 @@ impl TryFrom<ExportActiveBlock> for ActiveBlock {
             a_block.block.addresses_to_endorsements(&endorsement_ids)?;
         Ok(ActiveBlock {
             creator_address: Address::from_public_key(&a_block.block.header.content.creator),
-            block: a_block.block,
+            block: a_block.block.header.compute_block_id()?,
```

Comment:
that's a very heavy computation (hashing the whole block)

Reply:
We can remove it, by instead of using `try_into`, simply creating the block in block graph when bootstrapping, where the id is already available. This is more of a quick fix for the compilation.

Reply:
I really think objects should eventually carry their own ID